x-pack/plugins/timelines/server/search_strategy/index_fields/index.ts

```diff
@@ -63,9 +65,15 @@ export const requestIndexFieldSearch = async (
           });
           return get(searchResponse, 'body.hits.total.value', 0) > 0;
         } else {
-          return indexPatternsFetcher.getFieldsForWildcard({
-            pattern: index,
-          });
+          if (index.includes('.alerts-security.alerts-default')) {
```

Comment:
we should not hardcode the space id and the name but for now why not just doing `startWith(`.alerts-security.alerts`)` until we have a better way to get this information out of the rule registry.

Reply:
Updated! thanks a4e10f5